synapseclient/models/dataset.py

```diff
@@ -1213,12 +1066,10 @@ async def remove_item_async(
 
         Arguments:
             item: The Synapse ID or Entity to remove from the dataset
-            synapse_client: If not passed in and caching was not disabled by
-                `Synapse.allow_client_caching(False)` this will use the last created
-                instance from the Synapse class constructor.
+            synapse_client: The Synapse client to use. Defaults to None.
```

Comment:
```suggestion
            synapse_client: If not passed in and caching was not disabled by
                `Synapse.allow_client_caching(False)` this will use the last created
                instance from the Synapse class constructor.
```